Scripts/SumList/SumList.py

```diff
@@ -1,33 +1,37 @@
 from CommonServerPython import *
 
-value = demisto.args()["value"]
-
-if isinstance(value, int):
-    demisto.results(value)
-
-elif isinstance(value, list):
-    if len(value) == 1:
-        if isinstance(value, int):
-            demisto.results(value[0])
-        else:
-            try:
-                result = int(value[0])
-                demisto.results(result)
-            except TypeError:
-                return_error('The string does not represent a number.')
-    else:
+
+def sum_value(value):
+    if isinstance(value, int):
+        return value
+
+    if isinstance(value, str):
         try:
-            result = sum(value)
-            demisto.results(result)
-        except TypeError:
-            return_error('This transformer applies only to numbers.')
-
-elif isinstance(value, str):
-    try:
-        result = int(value)
-        demisto.results(result)
-    except TypeError:
-        return_error('The string does not represent a number.')
-
-else:
-    return_error('This transformer applies only to a list of numbers.')
+            value = [float(item) for item in value.split(',')]
+        except ValueError:
+            return 'error', 'error'
+
+    elif isinstance(value, list):
+        try:
+            value = [float(item) for item in value]
+        except ValueError:
+            return 'error', 'error'
+
+    result = sum(value)
+    if result == int(result):
```

Comment:
Why do we need this?

Reply:
Just to prettify the result, but it doesn't really matter